docs/en/connector/sink/Assert.md

```diff
@@ -0,0 +1,105 @@
+# Assert
+
+> # Sink plugin: Assert [Flink]
+
+## Description
+
+A flink sink plugin which can assert illegal data by user defined rules
+
+:::tip
+
+Engine Supported and plugin name
+
+* [x] Spark
```

Comment:
The connector developed based on the new API theoretically supports both spark and Flink. So I think this connector can run on Spark too.

Reply:
yes , I amend here, Thx